hdmi2usb/flash.txt

```diff
@@ -0,0 +1,7 @@
+cd ~/prebuilt
+
+python3 ../HDMI2USB-mode-switch/bin/download-prebuilt-firmware.py --platform opsis --rev v0.0.4-340-g3fb4e7c
+
+~/HDMI2USB-mode-switch/venv/bin/hdmi2usb-mode-switch -v --flash-image image-gateware+bios+firmware.v0.0.4-340-g3fb4e7c.opsis.hdmi2usb.lm32.bin
```

Comment:
Are these suppose to be hard coded?

Reply:
It's notes/example of how to flash which is done manually at a prompt because mode-switch needs to be watched for 181818

This will go away once it is automated.